### ISSUE TITLE
what is the ouput of CRF?

### ISSUE THREAD

**Issue by yiyi123123132** [keras-team/keras-contrib#543]
crf = CRF(len(tag2idx)) 
logits = crf(bert_output)

what is the ouput of the CRF,when I print the logits, it shows the dimension (?,?,tag2idx), why the CRF don't output the best sequence? (?,?,1)
